lib/src/main/java/io/ably/lib/http/TokenAuth.java

```diff
@@ -36,21 +36,21 @@ public void setTokenDetails(TokenDetails tokenDetails) {
 		this.encodedToken = Base64Coder.encodeString(tokenDetails.token).replace("=", "");
 	}
 
-	public TokenDetails authorise(AuthOptions options, TokenParams params) throws AblyException {
-		Log.i("TokenAuth.authorise()", "");
+	public TokenDetails authorize(AuthOptions options, TokenParams params) throws AblyException {
```

Comment:
I realise the spec doesn't require us to change this, but why not do it and keep things consistent?